### PR TITLE
Add phonegap nfc plugin typing

### PR DESCRIPTION
**Pull request by michaeldesigaud**

### CODE REVIEW

File: phonegap-nfc/phonegap-nfc.d.ts [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+ * Global object NFC.
+ */
+interface Window {
+    nfc: Nfc;
+    ndef: Ndef;
+    util:Util;
+    fireNfcTagEvent(event:TagEvent, tagAsJson:string):void;
+}
+
+interface Tag {
```
Comment (vvakame) [inline phonegap-nfc/phonegap-nfc.d.ts]: you should use the this pattern.
http://definitelytyped.org/guides/best-practices.html#ghost-modules

**Comment by vvakame** [comment]
please make `phonegap-nfc/phonegap-nfc-tests.ts` file. definition-tester can't pick up other names.

**Comment by michaeldesigaud** [comment]
I made the modifications

**Comment by vvakame** [comment]
please use `phonegap-nfc/phonegap-nfc-tests.ts`. not `phonegap-nfc/phonegap-nfc.tests.ts` :wink:

**Comment by michaeldesigaud** [comment]
fixed!

**Pull request merged**